src/generate.js

```diff
@@ -94,7 +94,7 @@ export const defaultSelectorHandlers = [
         }
         // Generate the styles normally, and then wrap them in the media query.
         const generated = generateSubtreeStyles(baseSelector);
-        return `${selector}{${generated}}`;
+        return [`${selector}{${generated}}`];
```

Comment:
Ah, I see. The alternative would be to do something like this on line 173:

```js
if (Array.isArray(result)) {
  generatedStyles.push(...result);
} else {
  generatedStyles.push(result);
}
```

I think making the breaking change is okay. @xymostech do you have any thoughts on this?